app/controllers/pipeline_jobs_controller.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+class PipelineJobsController < ApplicationController
+  before_action :find_pipeline
+  before_action :find_pipeline_job, only: %i[show cancel]
+
+  def index
+    @pipeline_jobs = paginate_and_filter_jobs(@pipeline.pipeline_jobs)
+  end
+
+  def show; end
+
+  def create
+    @pipeline_job = PipelineJob.new(pipeline_job_params)
+
+    @pipeline_job.save!
```

Comment:
Hey @paul-mesnilgrente, save! doesn't skip validations. It does the opposite as it will raise an error if the object fails to save.

https://www.linkedin.com/pulse/save-vs-ahtasham-naeem